bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -1,36 +1,38 @@
-from shared.gast_to_code.abstract_gast_to_code_converter import AbstractGastToCodeConverter
 import shared.gast_to_code.general_helpers as general_helpers
 import shared.gast_to_code.gast_to_code_router as router
+from shared.gast_to_code.error_handler import ErrorHandler
 
 
-class BashGastToCodeConverter(AbstractGastToCodeConverter):
+class BashGastToCodeConverter():
     name = "Bash"
     is_beta = True
     is_input_lang = False
     is_output_lang = True
 
-    def handle_bool(gast):
+    def __init__(self):
+        self.error_handler = ErrorHandler()
+
+    def get_error_handler(self):
+        return self.error_handler
+
+    def handle_bool(self, gast):
         return "Bash does not support booleans"
 
-    def handle_log_statement(gast):
+    def handle_log_statement(self, gast):
         return "echo"
 
-    def handle_func_call(gast):
-        if gast["value"][
-                "type"] == "logStatement" and general_helpers.arr_in_list(
-                    gast["args"]):
-            return "impossibleTranslationError: direct translation does not exist"  # TODO: streamline error messag
+    def handle_func_call(self, gast):
         return router.gast_to_code(gast["value"],
                                    "bash") + " " + bash_arg_helper(gast["args"])
 
-    def handle_arr(gast):
+    def handle_arr(self, gast):
         # This logic returns an error for nested arrays which are not supported in bash
         if general_helpers.arr_in_list(gast["elements"]):
-            return "impossibleTranslationError: direct translation does not exist"  # TODO: streamline error message as part of refactor
+            return self.error_handler.impossible_translation()
```

Comment:
Since this is V1, I am not adding in extra context yet. Want to just get boilerplate done and then add in more context later which will include links, etc. 

See [taiga#170](https://tree.taiga.io/project/jackdavidweber-cjs-capstone/us/170?milestone=269760)